waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java

```diff
@@ -118,6 +125,7 @@ public void init() throws Exception {
 
   @After
   public void destroy() throws Exception {
+    graphite.after();
```

Comment:
Maybe for a separate PR?

Reply:
yes was just running a quick test. (didn't work)